Components/Text Utilities/styles.css

```diff
@@ -0,0 +1,322 @@
+
+*
+{
+    font-family: 'Lato', sans-serif;
+}
+
+
+body
+{
+margin: 0;
+padding: 0;
+    
+
+}
+
+:root
+{
+    --primary-color: #63DCCD;
+    --secondary-color: #266350;
+}
+
+.logo-component
+{
+
+    height: 50px;
+    width: 60px;
+}
+
+.navigation-menu
+{
+    padding: 1.5rem 1rem;
+    /* width: 100%; */
+    display: flex;
+    align-items: center;
+    justify-content: space-evenly;
+    align-items: center;
+    /* border-bottom: 2px solid var(--primary-color); */
+}
+
+.left-side-nav
+{
+    display: flex;
+
+    justify-content: space-between;
+    align-items: center;
+}
+
+.right-side-nav
+{
+
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.sub-texts
+{
+    margin: 0.2rem;
+    color: var(--secondary-color);
+    text-decoration: none;
+    font-size: 1rem;
+    font-weight: 600;
+}
+
+
+.sub-texts:hover
+{
+    border-bottom: 2px solid var(--secondary-color);
+}
+
+.heading
+{
+    font-size: 2rem;
+    text-decoration: none;
+    font-weight: 700;
+    font-family: 'Lato', sans-serif;
```

Comment:
- Please check if this is needed? You already declared font family in upper section.